### PR TITLE
Update org-level readme

### PR DESCRIPTION
**Pull request by kylehoehns**
Most of this is just text/phrasing ripped from our website. I have a question out to Haley to verify things look good from her end but this is a start 😄

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
-# Welcome
+[![Source Allies](https://raw.githubusercontent.com/sourceallies/.github/main/profile/sa-header.png)](https://sourceallies.com/)
 
-[![Source Allies](https://raw.githubusercontent.com/sourceallies/.github/main/profile/sa-logo.svg)](https://sourceallies.com/)
+Source Allies is a technology services company, powered by people in slippers!
```
Comment (BenMcH) [inline profile/README.md]: "technology services company" is rather vague. Do we have a better definition? 

I wonder if we want to add in anything surrounding our goal of being the preferred place of employment for collaborative technologists seeking mastery.

Thank you for working on this!

Comment (kylehoehns) [reply profile/README.md]: I like that! Lemme swap that up and add it. I just copy/pasted from our front page but I can wordsmith a bit 😄

**Review by AndreasChristianson — APPROVED** [review]
lgtm. Good work!

**Comment by kylehoehns** [comment]
Ran this past Haley and she thought all looked good.

File: profile/README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
-# Welcome
+[![Source Allies](https://raw.githubusercontent.com/sourceallies/.github/main/profile/sa-header.png)](https://sourceallies.com/)
 
-[![Source Allies](https://raw.githubusercontent.com/sourceallies/.github/main/profile/sa-logo.svg)](https://sourceallies.com/)
+Source Allies is a technology services company, powered by people in slippers! Our goal is to be the preferred place of employment for collaborative technologists seeking mastery.
+
+## Join our team!
+
+Our partners tell us we act like *owners*, not renters. Our slippers symbolize this ownership mindset. Renters wear shoes at home, but an owner takes them off as they come through the door. We wear slippers in the office for the same reason. We are always looking for teammates and apprentices that align with our [expertise](https://www.sourceallies.com/what-we-do/#our-expertise) and [core values](https://www.sourceallies.com/#our-core-values).
```
Comment (mattcvincent) [inline profile/README.md:7]: Consider: For the same reason, we wear slippers both when working from home and from our offices.  Or something like that which acknowledges that we bring this mindset wherever we are.

Also, when we say "align", that reminds me of fit.  Focus on fit can result in homogeneity.  We want to hire teammates who can **enrich** and continue **improve** our culture, both strengthen and broaden our expertise, and embody our aspirational core values.

Comment (kylehoehns) [reply profile/README.md:7]: Great suggestions - I'll get that updated 👍

**Pull request merged**